tasks/gen/model.cr

```diff
@@ -0,0 +1,29 @@
+class Gen::Model < LuckyCli::Task
+  banner "Generate a model, query, and form"
+  getter io : IO = STDOUT
+
+  def call(@io : IO = STDOUT)
+    if ARGV.first?
+      render_templates
+    else
+      io.puts "Model name is required. Example: lucky gen.model User".colorize(:red)
+    end
+  end
+
+  private def render_templates
+    Lucky::ModelTemplate.new(model_name).render("./src/models/")
+    io.puts success_message("#{model_name}.cr", "./src/models/")
+    Lucky::FormTemplate.new(model_name).render("./src/forms/")
+    io.puts success_message("#{model_name}_form.cr", "./src/forms/")
+    Lucky::QueryTemplate.new(model_name).render("./src/queries/")
```

Comment:
Instead of having 3 separate template classes, I think you could have one `ModelTemplate` class that has all 3 files:

```
templates/
  model/
    src/models/{{model_name}}.cr
    src/forms/{{model_name}}_form.cr
    src/forms/{{model_name}}_query.cr
```

Reply:
It should definitely be possible with Teeplate.  My original intention was to have different templates that could be re-used for individual generator methods (e.g. `gen.form`), but I think that may have been a bit premature.  Using a single template here should make it easier to parse attribute arguments (e.g. `name:string` `email:string`) in a future PR.